content-src/components/DiscoveryStreamComponents/List/_List.scss

```diff
@@ -24,21 +50,35 @@ $item-line-height: 20;
   // regression detection?
   padding-inline-start: 0;
 
-  // "2/3 width layout"
-  .ds-column-5 &,
-  .ds-column-6 &,
-  .ds-column-7 &,
-  .ds-column-8 & {
-    grid-template-columns: repeat(2, 1fr);
+  &:not(.ds-list-full-width) {
+    @include set-item-sizes($item-font-size, $item-line-height, $item-image-size);
+
+    // "2/3 width layout"
+    .ds-column-5 &,
+    .ds-column-6 &,
+    .ds-column-7 &,
+    .ds-column-8 & {
+      grid-template-columns: repeat(2, 1fr);
+    }
+
+    // "Full width layout"
+    .ds-column-9 &,
+    .ds-column-10 &,
+    .ds-column-11 &,
+    .ds-column-12 & {
+      grid-template-columns: repeat(3, 1fr);
+      grid-row-gap: 18px;
+    }
+
+    .ds-list-item-excerpt {
+      display: none;
```

Comment:
Could we not render this instead, since `fullWidth` is a prop?

Reply:
There had been requests to avoid conditionally rendering content to allow remote css to unhide if necessary.